### PR TITLE
feat: docs for code autoevals

### PR DESCRIPTION
**Pull request by jakerachleff**

### CODE REVIEW

File: versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Set up the auto-evaluator
+
+Go to the datasets UI, select your dataset, and click the Auto Evaluators button. When prompted,
+select the `Custom Code` type evaluator.
+
+## Write your evaluation function
+
+:::note
+Custom Code evaluators use pyodide sandboxes under the hood to ensure security for our customers. You are
+restricted to import only their [supported packages](https://pyodide.org/en/stable/usage/packages-in-pyodide.html).
```
Comment (akira) [inline versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx:15]: FYI we don't allow all packages, only a few white listed ones

Comment (akira) [reply versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx:15]: These are ones we include:
```
  numpy: "numpy",
  pandas: "pandas",
  jsonschema: "jsonschema",
  scipy: "scipy",
  sklearn: "scikit-learn",
```

Comment (jakerachleff) [reply versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx:15]: Oh where do we document this?

Comment (akira) [reply versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx]: Sorry to clarify, we include the standard python libraries, but for pip packages those are the ones included.  I would list as:

* numpy
* pandas
* jsonschema
* scipy
* scikit-learn

**Review by akira — APPROVED** [review]

**Pull request merged**